force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1462,6 +1462,22 @@
         <shortDescription>Label for the Require Preferred Phone setting</shortDescription>
         <value>Require Preferred Phone</value>
     </labels>
+    <labels>
+        <fullName>stgDuplicateRelationships</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Label for the Relationship Duplicates Settings</shortDescription>
+        <value>Duplicate Relationships</value>
```

Comment:
Suggest we would name this as Duplicate Relationships Prevention. Pending doc's response.